subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -26,6 +26,11 @@ LOG_MODULE_REGISTER(nrf_cloud_transport, CONFIG_NRF_CLOUD_LOG_LEVEL);
 
 #if defined(CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES)
 #include CONFIG_NRF_CLOUD_CERTIFICATES_FILE
+#if defined(CONFIG_NRF_INBUILT_KEY)
+#include <nrf_inbuilt_key.h>
+#else
+#error "Missing CONFIG_NRF_INBUILT_KEY"
```

Comment:
This will fail for `qemu_x86` target.

Reply:
For reference, I got the inspiration from here: 
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/samples/nrf9160/lwm2m_client/src/main.c#L29-L31

Should we instead silently ignore and let compiler detect missing symbols?

Reply:
`lwm2m_client` does not support qemu, `asset_tracker` does. I'm not sure why doesn't CI build for qemu targets, as several samples have qemu configs.

Reply:
Ah, ok. I'll remove both instances of this.